src/app/lib/mainsail/client.service.ts

```diff
@@ -64,6 +63,20 @@ export class ClientService {
 		);
 	}
 
+	public async unconfirmedTransactions(
```

Comment:
added this to the client service so we can rely on it for fetching the unconfirmed transactions in a single place. It's using the `ConfirmedTransactionData` as it overlaps with the unconfirmed ones. We can discuss how/if we want to address this further